internal/utils/access_token.go

```diff
@@ -84,6 +84,23 @@ func fallbackSaveToken(accessToken string, fsys afero.Fs) error {
 	return afero.WriteFile(fsys, path, []byte(accessToken), 0600)
 }
 
+func DeleteAccessToken(fsys afero.Fs) error {
+	// Delete from native credentials store
+	if err := credentials.Delete(AccessTokenKey); err == nil {
+		return nil
+	}
+	// Fallback to token file
+	return fallbackDeleteToken(fsys)
+}
+
+func fallbackDeleteToken(fsys afero.Fs) error {
+	path, err := getAccessTokenPath()
```

Comment:
It should not fail when there's no access token found. When `NotFound` error is returned, `logout` command should just print `You were not logged in, nothing to do.` or similar message. Now it'll fail with:
```
remove /Users/kamilogorek/.supabase/access-token: no such file or directory
```
When called multiple times.

Reply:
Thanks. I checked if the error returned by `Remove` is ENOENT, This is OK for OS fs implementation in afero. But I didn't find a common error or check function in afero that would express different fs NotFound error.